src/Compilers/CSharp/Test/Syntax/Parsing/SuppressNullableWarningExpressionParsingTests.cs

```diff
@@ -596,15 +596,15 @@ public void ConditionalAccess_07()
 
             N(SyntaxKind.ConditionalExpression);
             {
-                N(SyntaxKind.ConditionalAccessExpression);
+                N(SyntaxKind.SuppressNullableWarningExpression);
```

Comment:
This baseline changed because we are checking for whether the `!` is followed by a *dependent-access* in a more uniform way. In this case, we realize the trailing `?` can't start a new conditional-access, so we treat the `!`-expr as the parent of the current conditional-access instead.

It's possible that some valid case exists involving this production which this is changing. However it does necessarily involve a rather convoluted mix of `!` and `?`s.